WooCommerce/WooCommerceTests/ViewRelated/BottomSheet/BottomSheetPresenterTests.swift

```diff
@@ -0,0 +1,55 @@
+import XCTest
+@testable import WooCommerce
+
+final class BottomSheetPresenterTests: XCTestCase {
+    func test_sheet_customizations_are_set_to_sheetPresentationController() throws {
+        // Given
+        let presenter = BottomSheetPresenter { bottomSheet in
+            var sheet = bottomSheet
+            sheet.prefersEdgeAttachedInCompactHeight = false
+            sheet.largestUndimmedDetentIdentifier = nil
+            sheet.prefersGrabberVisible = false
+            sheet.detents = [.large()]
+        }
+        let viewController = UIViewController()
+
+        // When
+        presenter.present(viewController, from: .init())
+
+        // Then
+        XCTAssertTrue(viewController.sheetPresentationController?.prefersEdgeAttachedInCompactHeight == false)
+        XCTAssertNil(viewController.sheetPresentationController?.largestUndimmedDetentIdentifier)
+        XCTAssertTrue(viewController.sheetPresentationController?.prefersGrabberVisible == false)
+        XCTAssertEqual(viewController.sheetPresentationController?.detents, [.large()])
```

Comment:
Nit: should we unwrap the sheet to avoid testing optional?

```suggestion
        let sheet = try XCTUnwrap(viewController.sheetPresentationController)
        XCTAssertFalse(sheet.prefersEdgeAttachedInCompactHeight)
        XCTAssertNil(sheet.largestUndimmedDetentIdentifier)
        XCTAssertFalse(sheet.prefersGrabberVisible)
        XCTAssertEqual(sheet.detents, [.large()])
```

Reply:
Updated in https://github.com/woocommerce/woocommerce-ios/pull/9552/commits/984110a24a1f880610653401700aa7007a9c0f73